qualtran/bloqs/factoring/ecc/ec_add.py

```diff
@@ -33,6 +37,8 @@ class ECAdd(Bloq):
     Args:
         n: The bitsize of the two registers storing the elliptic curve point
         mod: The modulus of the field in which we do the addition.
+        curve_a: The $a$ coefficient of an elliptic curve given in the standard form of
```

Comment:
does this actually affect the decomposition?

This is a little confusing. In litinski they use (x, y) + (a, b) for the curve points and c1, c2 for the curve coefficients. Other references use (x1, y1) + (x2, y2) and a, b for the curve points. Maybe we should switch to using this second scheme.

Reply:
I no longer need curve_a for the classical simulation, I can get it from a, b, and lambda_r. I think a and b for the curve coefficients is much more clear than c1, c2. I can swap from (x,y) + (a,b) to (x1,y1) + (x2,y2), but I think (x, y) + (a, b) is easier to understand. I think since we use curve_a instead of just a it is clear that it is different. I'll defer judgement to you on the naming convention and I can change it if you think that's better.

Reply:
point taken on `curve_a` but we'll probably want to write latex expressions in some places and then $a$ will be ambiguous.